cs3api4lab/tests/test_cs3_file_api.py

```diff
@@ -165,6 +161,26 @@ def test_remove_file(self):
         with self.assertRaises(IOError):
             self.storage.stat(fileid, self.userid, self.endpoint)
 
+    def test_read_directory(self):
+
+        fileid = "/"
+        read_directory = self.storage.read_directory(fileid, self.userid, self.endpoint)
+        self.assertIsNotNone(read_directory[0])
+        self.assertIsNotNone(read_directory[0].path)
+
+    def test_move_file(self):
+
+        src_id = "/file_to_rename.txt"
+        buffer = b"ebe5tresbsrdthbrdhvdtr"
+
+        dest_id = "/file_after_rename.txt"
+
+        self.storage.write_file(src_id, self.userid, buffer, self.endpoint)
+        self.storage.move(src_id, dest_id, self.userid, self.endpoint)
+
```

Comment:
shouldn't we stat here to check the file exists where it should?

Reply:
Not because if the save fails, an error will occur and the test will not be performed, and after performing the test, I delete the added files.

Reply:
An error will occur *IF* it's well implemented, right? So... shouldn't we test it? :)